### DISCUSSION TITLE
How can I "fake" the target_os?

### DISCUSSION THREAD

**Discussion by psionic12** [intellij-rust/intellij-rust#8915] [Q&A] [answered]
In my case, I have some codes like this:
```
#[cfg(any(target_os = "android"))]
// some codes
#[cfg(any(target_os = "macos", target_os = "ios"))]
// some codes
```

The rust plugin runs on Android Studio which runs on a macOS, so the plugin considered the arget_os to be "macos". But when I'm reading the code, I do want the `target_os` to be "android", to get the right code navigation.

Anyway to that?

**Comment by Undin** [comment, marked as answer]
You can add `.cargo/config.toml` file to your cargo project and specify the proper build target for Android, for example

```toml
[build]
target = "aarch64-linux-android"
```

and after that, you need to reload a project model via `Refresh Cargo Projects` action to apply your changes
<img width="310" alt="image" src="https://user-images.githubusercontent.com/2539310/172798542-9cf1f3da-c177-41ad-b935-5896b3311bcd.png">
